Blazor.sln

```diff
@@ -12,16 +12,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "mono", "mono", "{7B5CAAB1-A
 		src\mono\mono.targets = src\mono\mono.targets
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Blazor.Mono", "src\Microsoft.AspNetCore.Blazor.Mono\Microsoft.AspNetCore.Blazor.Mono.csproj", "{39FEC72D-AF52-47A3-B63D-7BF0E4335248}"
```

Comment:
Is this destined for `dev` or `feature/0.1.0`?

Reply:
Dev then 0.1.0? /cc: @SteveSandersonMS?

Reply:
I think we're ready to merge `0.1.0` back into `dev`, right @rynowak? I'm not aware we have any remaining reason to keep it separate.

If that's the case then @javiercn it's up to you: would you prefer to merge `0.1.0` into `dev` first and then you rebase onto that, or would you prefer to merge this PR into `dev` first and then you rebase `0.1.0` onto the new `dev`?